judgments/tests/test_detail.py

```diff
@@ -492,8 +512,8 @@ def test_document_headings_when_press_summary(
         self, mock_get_document_by_uri, mock_get_pdf_size
     ):
         """
-        GIVEN a press summary and related judgment
-        WHEN a request is made with the press summary URI
+        WHEN a request is made with to a document detail page
```

Comment:
could these 2 lines be swapped to be consistent with the Given-When-Then syntax?